data/summary/types/summary_test.go

```diff
@@ -84,6 +82,15 @@ func ExpectedAverage(windowSize int, hoursAdded int, newAvg float64, oldAvg floa
 	return (oldAvgTotal + newAvgTotal) / float64(windowSize)
 }
 
+func ConvertToIntArray[T types.RecordTypes](arr []*T) []interface{} {
```

Comment:
Nit: I don't see any ints here... Did you just use Int as an abbreviation for interface{}? If so, consider maybe Any instead, as it at least doesn't clash with an existing (and common!) type.

Reply:
yeah... it stands for "interface", it is a bit weird.